welcome/index.adoc

```diff
@@ -61,13 +61,27 @@ does not have full access to the internet, you can
 xref:../installing/install_config/installing-restricted-networks-preparations.adoc#installing-restricted-networks-preparations[mirror the {product-title} installation images]
 and install a cluster in a restricted network.
 
-- **xref:../installing/installing-gather-logs.adoc#[Check installation logs]**: Access installation logs to evaluate issues that occur during {product-title} {product-version} installation.
+- **Install a cluster in an existing network**: If you use an existing VPC in
+xref:../installing/installing_aws/installing-aws-vpc.adoc#installing-aws-vpc[AWS] or
+xref:../installing/installing_gcp/installing-gcp-vpc.adoc#installing-gcp-vpc[GCP] or an existing
+xref:../installing/installing_azure/installing-azure-vnet.adoc#installing-azure-vnet[VNet]
+on Azure, you can install a cluster 
+
+- **Install a private cluster**: If your cluster does not require external
+internet access, you can install a private cluster on
+xref:../installing/installing_aws/installing-aws-private.adoc#installing-aws-private[AWS],
+xref:../installing/installing_azure/installing-azure-private.adoc#installing-aws-private[Azure], or
+xref:../installing/installing_gcp/installing-gcp-private.adoc#installing-gcp-private[GCP].
+You still require internet access to access the cloud APIs and installation
```

Comment:
Or

s/You still require internet access to access the cloud APIs and installation media./Internet access is still required to access the cloud APIs and installation media./